ledger-service/http-json/src/failurelib/scala/http/FailureTests.scala

```diff
@@ -100,6 +107,9 @@ abstract class FailureTests
   "Command submission timeouts" in withHttpService { (uri, encoder, _, client) =>
     import json.JsonProtocol._
     for {
+      _ <- Future.sequence(packageFiles.map { dar =>
+        client.packageManagementClient.uploadDarFile(ByteString.readFrom(new FileInputStream(dar)))
+      })
```

Comment:
This should be in `HttpFailureTestFixture#withHttpService`, which factors all the copies you've added in this file.